src/actions/l10n.js

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+import type { Action, ThunkAction, Localization } from 'firefox-profiler/types';
+import {
+  fetchMessages,
+  lazilyParsedBundles,
+} from 'firefox-profiler/app-logic/l10n-bundling';
+import { negotiateLanguages } from '@fluent/langneg';
+import { ReactLocalization } from '@fluent/react';
+/**
+ * Notify that translations for the UI are being fetched.
+ */
+export function requestL10n(): Action {
+  return {
+    type: 'REQUEST_L10N',
+  };
+}
+
+/**
+ * Receive translations for a locale.
+ */
+export function receiveL10n(localization: Localization): Action {
+  return {
+    type: 'RECEIVE_L10N',
+    localization: localization,
+  };
+}
+
+export const AVAILABLE_LOCALES: Array<string> = ['en-US', 'fr'];
```

Comment:
Also, for now, I think it's okay to keep an array of available locales but I guess it would be good to automate this step somehow (like getting all the directories inside the 'locales' folder and creating this array instead). Or we can document this step so we don't forget this in the future.